src/classes/CW_AudienceGroupControllerTest.cls

```diff
@@ -0,0 +1,333 @@
+/**
+* File Name   :    CW_AudienceGroupControllerTest.cls
+* Description :    B-110228/B-110230/B-109518
+*                  CIM, BI, Marketer must be able to View,Edit,Delete audience groups 
+*				   that are linked to the campaign
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 03/04/2014 Cristina P		Initial
+ */
+@isTest
+private class CW_AudienceGroupControllerTest 
+{
+
+	public static CW_Campaign_Navigation_Settings__c 	cns1;
+	public static CW_Campaign_Navigation_Settings__c 	cns2;
+	
+	public static CW_CampaignListSetting__c		audienceGroupPageSizeRecord;
+	
+	public static void initCustomSettings()
+	{
+		cns1 = TestDataUtil.createCampaignNavigationSettings(Page.CW_Campaign_Information,1,false);
+    	cns2 = TestDataUtil.createCampaignNavigationSettings(Page.CW_Audience,2,false);
+    	insert new List<CW_Campaign_Navigation_Settings__c>{cns1,cns2};
+    	
+    	audienceGroupPageSizeRecord = new CW_CampaignListSetting__c(Name=ConstantUtil.AUDIENCE_GROUP_PAGE_SIZE,
+    											 Integer_Value__c=10);
+    	insert audienceGroupPageSizeRecord;
+	}
+	
+	static testMethod void standardSetControllerTest() 
```

Comment:
Style issue: test methods should be annotated with @IsTest for better readibility